docs/source/training.rst

```diff
@@ -97,7 +97,7 @@ Trains a diffusion model to insert glasses onto faces.
 
 .. code:: bash
 
-   python3 train.py --dataroot noglasses2glasses_ffhq --checkpoints_dir ./checkpoints --name glasses2noglasses --output_display_env glasses2noglasses --config_json examples/example_ddpm_glasses2noglasses.json
+   python3 train.py --dataroot /path/to/data/noglasses2glasses_ffhq --checkpoints_dir /path/to/checkpoints --name glasses2noglasses --output_display_env glasses2noglasses --config_json examples/example_ddpm_noglasses2glasses.json
```

Comment:
⚠️  the `--name` and `--output_display_env` are glasses2noglasses but this section is about glasses **insertion**, not removal. Should be noglasses2glasses instead